factory/processComponents.go

```diff
@@ -478,6 +482,18 @@ func (pcf *processComponentsFactory) Create() (*processComponents, error) {
 		return nil, err
 	}
 
+	pcf.esdtNftStorage, err = vmcommonBuiltInFunctions.NewESDTDataStorage(vmcommonBuiltInFunctions.ArgsNewESDTDataStorage{
```

Comment:
could have had an arg variable here, it is harder to read like this

Reply:
done